crates/ruff_workspace/src/options.rs

```diff
@@ -1666,11 +1669,19 @@ impl Flake8ImportConventionsOptions {
             );
         }
 
-        flake8_import_conventions::settings::Settings {
-            aliases,
+        let normalized_aliases: FxHashMap<String, String> = aliases
+            .into_iter()
+            .map(|(module, alias)| {
+                let normalized_alias = alias.nfkc().collect::<String>();
+                (module, normalized_alias)
+            })
+            .collect();
```

Comment:
Oh. I don't think we need to use `try_into_settings` if we only do the normalization here. 

What I had in mind is to error if the user provides an alias that normalizes to `__debug__`. Or are there cases where we use `aliases` where `__debug__` should be allowed?

Reply:
I don't think `__debug__` is ever valid as an alias. Is this the right idea?

```rust
let mut normalized_aliases: FxHashMap<String, String> = FxHashMap::default();
for (module, alias) in aliases {
    let normalized_alias = alias.nfkc().collect::<String>();
    if normalized_alias == "__debug__" {
        anyhow::bail!(
            "Invalid alias for module '{module}': alias normalizes to '__debug__', which is not allowed."
        );
    }
    normalized_aliases.insert(module, normalized_alias);
}
```

Reply:
Yes, I think that would be the ideal user experience (telling them that the configuration is invalid). 

An alternative is to use `warn_user_once` to log a warning instead (and not adding the alias)

I think either's fine